pkg/checks/kubeconform/validate.go

```diff
@@ -60,12 +44,14 @@ func getSchemaLocations(ctx context.Context, ctr container.Container, tempRepoPa
 
 		location += "{{ .NormalizedKubernetesVersion }}/{{ .ResourceKind }}{{ .KindSuffix }}.json"
```

Comment:
We clone (potentially) all the repos in ./cmd/controller.go, lines 73 and 78. Each repository that we encounter in those lists is cloned, and their url is replaced with the local path to their cloned sources.

Reply:
I see that the repos are cloned here: https://github.com/zapier/kubechecks/blob/f413659bf66f2cb3aea34c77fe650a8b9a39102f/cmd/controller.go#L77C36-L77C52

What I don't see is where the clone is used. 
https://github.com/zapier/kubechecks/blob/f413659bf66f2cb3aea34c77fe650a8b9a39102f/pkg/checks/kubeconform/validate.go#L60

I'm asking because, my local setup is failing to use the schemas in the remote repo:
```
1:16PM DBG schema locations: default, https://github.com/mensaah/kubechecks-schemas/{{ .NormalizedKubernetesVersion }}/{{ .ResourceKind }}{{ .KindSuffix }}.json
```

This is outside the scope of this PR. I'll try and see if I can resolve it and create an independent PR/issue